lib/releaselib.sh

```diff
@@ -1145,6 +1145,12 @@ release::send_announcement () {
   ((FLAGS_nomock)) || mailto=$GCP_USER
   mailto=${FLAGS_mailto:-$mailto}
 
+  if ((FLAGS_print)); then
+    echo "You can find the Announcement file here: $archive_root/announcement.html"
+    echo "To copy to your local machine use: $GSUTIL cp $archive_root/announcement.html ."
+    return 0
```

Comment:
[I assume that this results in an `exit 0` and this is intended]

Reply:
if you look the others in the same function it uses `return` using the same here as well